### PR TITLE
JAX CKB link in cancer variant page

### LINKED ISSUE #3540: JAX CKB
Linkout for cancer variants: https://ckb.jax.org. Looks neat, though freemium: limited to a core set of 50 genes; the rest are for a fee. I don't get the best signals here, but better than some other sources at least. We might link out to the core part anyway and ask if local users like it and if they might want (to buy) the full one. Thoughts?

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3540 .

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. Test it locally with demo cancer case and TP53 variant or on cg-vm1

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/links.py [hunk 1]
```diff
@@ -50,7 +52,9 @@ def add_gene_links(gene_obj, build=37):
     gene_obj["exac_link"] = exac(ensembl_id)
     gene_obj["gnomad_link"] = gnomad(ensembl_id, build)
     # Add links that use entrez_id
-    gene_obj["entrez_link"] = entrez(gene_obj.get("entrez_id"))
```
Comment (northwestwitch) [inline scout/server/links.py:53]: This wasn't working

File: scout/server/links.py [hunk 2]
```diff
@@ -50,7 +49,9 @@ def add_gene_links(gene_obj, build=37):
...
+    entrez_id = gene_obj.get("common", {}).get("entrez_id")
```
Comment (dnil) [inline scout/server/links.py:52]: 👍

File: tests/server/test_links.py [hunk 3]
```diff
@@ -83,6 +90,13 @@ def test_mycancergenome_link():
     assert link is not None
 
 
+def test_ckb_link():
```
Comment (dnil) [inline tests/server/test_links.py:93]: 👍

File: scout/server/links.py [hunk 4]
```diff
@@ -100,6 +101,13 @@ def gnomad_str_gene(hgnc_symbol):
     return link.format(hgnc_symbol)
 
 
+def ckb_gene(entrez_id):
```
Comment (dnil) [inline scout/server/links.py:104]: One could consider a ckb_variant as well, but its based on like protein positions on an reference protein product given on the site... Im sure they are pretty stable for the core set of genes, but still. Not super solid practice. Gene only is a good call until someone says they are would actually really want it.

**Review by dnil — APPROVED** [review]

File: scout/server/links.py (hunk 4 above)
Comment (northwestwitch) [reply scout/server/links.py:104]: Agreed!

**Pull request merged**